packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/AssetViewOnly/AssetViewOnly.tsx

```diff
@@ -0,0 +1,451 @@
+import CoinDisplay from 'components/Display/CoinDisplay'
+import { Flex } from 'components/Flex'
+import { actions, selectors } from 'data'
+import { NftOrderStepEnum } from 'data/components/nfts/types'
+import { RootState } from 'data/rootReducer'
+import { isMobile, media, useMedia } from 'services/styles'
+
+import {
+  AssetName,
+  CollectionName,
+  CustomLink,
+  LeftColWrapper,
+  RightColWrapper,
+  Top,
+  Trait,
+  TraitsWrapper,
+  Wrapper
+} from '../Asset/components'
+import NftCollectionImage from '../components/NftCollectionImage'
+
+const AssetImageContainer = styled.div`
+  position: relative;
+  border-radius: 16px;
+  ${media.tablet`
+    padding: 0;
+  `};
+`
+
+const AssetImg = styled.img`
+  object-fit: cover;
+  height: 115vh;
+  ${media.tablet`
+    height: 100%;
+  `};
+`
+
+const Description = styled.div`
+  margin: 1em 0em;
+  padding: 1em;
+  background: rgb(240, 242, 247, 0.3);
+  border: 1px solid ${(props) => props.theme.greyFade000};
+  border-radius: 16px;
+`
+
+const CreatorOwnerAddress = styled.div`
+  font-size: 16px;
+  color: ${colors.grey700};
+  display: flex;
+`
+
+const CreatorOwnerAddressLinkText = styled(CreatorOwnerAddress)`
+  color: ${colors.blue600};
+  font-weight: 600;
+`
+
+const TokenDisplay = styled(Text)`
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  width: 200px;
+  text-align: right;
+`
+const DropdownPadding = styled.div`
+  padding-bottom: 1em;
+`
+
+const Detail = styled(Text)`
+  display: flex;
+  justify-content: space-between;
+  color: ${colors.grey900};
+  padding: 1em 1.5em;
+  border-bottom: 1px solid ${colors.grey000};
+  font-size: 16px;
+  font-weight: 500;
+  &:last-child {
+    border-bottom: none;
+  }
+`
+
+const ShadowTag = styled.div`
+  background: ${colors.white900};
+  box-shadow: 0px 4px 16px rgba(5, 24, 61, 0.1);
+  border-radius: 16px;
+  padding: 6px 6px;
+  width: fit-content;
+`
+
+const CollectionHeader = styled.div`
+  ${media.atLeastTablet`
+    justify-content: left;
+    // gap: 24px;
+  `};
+  display: flex;
+  margin-top: 2px;
+  width: 100%;
+  justify-content: space-between;
+  ${media.mobile`
+    padding-top: 32px;
+  `};
+`
+
+const GradientCoinDisplay = styled(CoinDisplay)`
+  background: linear-gradient(92.99deg, #7663ff 0.55%, #0069fc 98.76%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+`
+
+const ItemDetails = styled.div`
+  padding: 32px 0em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const MarginButton = styled(Button)`
+  margin: 1em 0;
+`
+
+const DetailsAndOffers = styled.div``
+
+const NftAssetViewOnly: React.FC<Props> = ({
+  nftOwnerAssets,
+  nftsActions,
+  routerActions,
+  ...rest
+}) => {
+  const goToView = () => {
+    routerActions.push(`/nfts/view`)
+  }
+  const { contract, id } = rest.computedMatch.params
+  const [moreAssetToggle, setMoreAssetToggle] = useState(true)
+
+  return (
+    <>
+      {nftOwnerAssets.cata({
```

Comment:
not a show stopper, but I highly suggest checking out the `useRemote()` hook so you don't have to do this `cata` stuff anymore.